index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html  ng-app="myApp">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        
+        <title>Library Information System Angular 1</title>
+        
+        <!-- Bootstrap CSS CDN -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-grid/4.0.11/ui-grid.css" />
+        <!-- Our Custom CSS -->
+        <link rel="stylesheet" href="styles/style.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    </head>
+    
+    <body>
+       
+       <div data-ng-controller="loginCtrl"> 
+
+           <div class="wrapper">
+               <nav ng-include="'views/nav.html'"></nav>
+                   
+               <div ui-view></div>
+           </div>
+       </div>
+       
+        <!-- Angular CDN -->
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular-route.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.4.3/angular-ui-router.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-grid/4.0.11/ui-grid.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.0/angular-animate.js"></script>
+
+        <script type="text/javascript" src="js\app.js"></script>
+        <script src="js/controllers/home.js"></script>
```

Comment:
I don't know if it strictly needed anymore, but for backwards compatibility can you add `type="text/javascript"` to all script tags please?